ui/components/component-library/popover/popover.test.tsx

```diff
@@ -0,0 +1,201 @@
+          {PopoverPosition.TopStart}
+        </Popover>
+
+        <Popover
+          isOpen={true}
+          data-testid={PopoverPosition.TopEnd}
+          position={PopoverPosition.TopEnd}
+        >
+          {PopoverPosition.TopEnd}
+        </Popover>
+
+        <Popover
+          isOpen={true}
+          data-testid={PopoverPosition.Right}
+          position={PopoverPosition.Right}
+        >
+          {PopoverPosition.Right}
+        </Popover>
+
+        <Popover
+          isOpen={true}
+          data-testid={PopoverPosition.RightStart}
+          position={PopoverPosition.RightStart}
+        >
+          {PopoverPosition.RightStart}
+        </Popover>
+
+        <Popover
+          isOpen={true}
+          data-testid={PopoverPosition.RightEnd}
+          position={PopoverPosition.RightEnd}
+        >
+          {PopoverPosition.RightEnd}
+        </Popover>
+
+        <Popover
+          isOpen={true}
+          data-testid={PopoverPosition.Bottom}
+          position={PopoverPosition.Bottom}
+        >
+          {PopoverPosition.Bottom}
+        </Popover>
+
+        <Popover
+          isOpen={true}
+          data-testid={PopoverPosition.BottomStart}
+          position={PopoverPosition.BottomStart}
+        >
+          {PopoverPosition.BottomStart}
+        </Popover>
+
+        <Popover
+          isOpen={true}
+          data-testid={PopoverPosition.BottomEnd}
+          position={PopoverPosition.BottomEnd}
+        >
+          {PopoverPosition.BottomEnd}
+        </Popover>
+
+        <Popover
+          isOpen={true}
+          data-testid={PopoverPosition.Left}
+          position={PopoverPosition.Left}
+        >
+          {PopoverPosition.Left}
+        </Popover>
+
+        <Popover
+          isOpen={true}
+          data-testid={PopoverPosition.LeftStart}
+          position={PopoverPosition.LeftStart}
+        >
+          {PopoverPosition.LeftStart}
+        </Popover>
+
+        <Popover
+          isOpen={true}
+          data-testid={PopoverPosition.LeftEnd}
+          position={PopoverPosition.LeftEnd}
+        >
+          {PopoverPosition.LeftEnd}
+        </Popover>
+      </>,
+    );
+
+    expect(getByText(PopoverPosition.Auto)).toBeDefined();
+    expect(getByText(PopoverPosition.Top)).toBeDefined();
+    expect(getByText(PopoverPosition.TopStart)).toBeDefined();
+    expect(getByText(PopoverPosition.TopEnd)).toBeDefined();
+    expect(getByText(PopoverPosition.Right)).toBeDefined();
+    expect(getByText(PopoverPosition.RightStart)).toBeDefined();
+    expect(getByText(PopoverPosition.RightEnd)).toBeDefined();
+    expect(getByText(PopoverPosition.Bottom)).toBeDefined();
+    expect(getByText(PopoverPosition.BottomStart)).toBeDefined();
+    expect(getByText(PopoverPosition.BottomEnd)).toBeDefined();
+    expect(getByText(PopoverPosition.Left)).toBeDefined();
+    expect(getByText(PopoverPosition.LeftStart)).toBeDefined();
+    expect(getByText(PopoverPosition.LeftEnd)).toBeDefined();
```

Comment:
This looks as though it's testing if the `PopoverPostion` enum text exists in the document it doesn't seem to be checking the position of the `Popover` in any way? Can we at least check for the classNames or data attributes?